python/kernmlops/data_schema/generic_table.py

```diff
@@ -0,0 +1,43 @@
+def class_generic_table(name: str, tableName: str):
+    return f"""
+import polars as pl
+from data_schema.schema import (
+    CollectionGraph,
+    CollectionTable,
+)
+
+class {tableName}(CollectionTable):
+
+    @classmethod
+    def name(cls) -> str:
+        return "{name}"
+
+    @classmethod
+    def schema(cls) -> pl.Schema:
+        return pl.Schema()
```

Comment:
you may want to expand this in the future to support custom schemas

Reply:
Very much agree, I think that belongs in another pr.